fedimint-cli/src/lib.rs

```diff
@@ -1249,10 +1249,13 @@ impl FedimintCli {
                 ))
             }
             Command::Completion { shell } => {
+                let bin_name = std::env::args()
+                    .next()
+                    .expect("Binary name is always provided if we get this far");
```

Comment:
Actually, this might mess up when the full path is specified :/

Reply:
Might need to parse a path and take the last element?